src/surface/rov_flir/launch/flir_launch.py

```diff
@@ -20,20 +20,22 @@ def generate_launch_description() -> LaunchDescription:
 
     parameters = {
         'debug': False,
-        'acquisition_timeout': 0.5,
         'compute_brightness': False,
         'adjust_timestamp': True,
         'dump_node_map': False,
         # set parameters defined in blackfly_s.yaml
         'gain_auto': 'Continuous',
-        'pixel_format': 'BayerRG8',
+        # 'pixel_format': 'BayerRG8',
         'exposure_auto': 'Continuous',
+        # to use a user set, do this:
+        # 'user_set_selector': 'UserSet0',
+        # 'user_set_load': 'Yes',
         # These are useful for GigE cameras
-        'device_link_throughput_limit': 125000000,
-        'gev_scps_packet_size': 9000,
+        # 'device_link_throughput_limit': 380000000,
+        # 'gev_scps_packet_size': 9000,
         # ---- to reduce the sensor width and shift the crop
-        'image_width': 720,
-        'image_height': 540,
+        # 'image_width': 1408,
+        # 'image_height': 1080,
```

Comment:
Group together and add a comment explaining the lines that control whether jumbo packets are enabled